modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -101,6 +101,70 @@ Previously, the three dots menu button for category and template in the **Templa
 
 In {productname} 7.0, the three dot menu buttons for categories and templates in the Templates dialog now have informative tooltips. This clarifies their purpose for all users, enhancing discoverability and improving the overall experience, especially for those relying on assistive technologies.
 
+=== Enhanced Code Editor
+
+The {productname} 7.0 release includes an accompanying release of the **Enhanced Code Editor** premium plugin.
+
+**Enhanced Code Editor** includes the following fixes, additions, and improvements.
+
+==== Updated CodeMirror to version 6.
+// #TINY-9205
+
+{productname} 7.0 introduces CodeMirror 6.
+
+With this update, users have gained access to improved features "to be expanded on in future releases" and a more responsive editing experience.
+
+During the integration process, {productname} made necessary adjustments to ensure compatibility between {productname} 7.0 and CodeMirror 6. Specifically, refining our CSS reset rules to accommodate the dynamically generated styles of CodeMirror 6.
+
+More information about CodeMirror 6 will be included in future patch releases.
+
+=== Moved CodeMirror 6 inside ShadowDom.
+// #TINY-10561
+
+With the release of {productname} 7.0, CodeMirror 6 has been integrated into the shadow DOM. This update from CodeMirror 5 to v6 should have no impact on customers utilizing the Enhanced Code Editor feature.
```

Comment:
Talking about shadow dom is too literal, as was mentioned when that was put into the changelog.
```suggestion
During the integration process, {productname} made necessary adjustments to ensure compatibility between {productname} 7.0 and CodeMirror 6. Specifically, our CSS reset rules were refined to accommodate the dynamically generated styles of CodeMirror 6 and as a result the `codemirror.css` file was removed from the plugin deployment.

This update from CodeMirror 5 to 6 should have no impact on customers utilizing the Enhanced Code Editor feature. More information about CodeMirror 6 will be included in future patch releases.
```